### DISCUSSION TITLE
如何写规则让同一域名按情况走不同的节点

### DISCUSSION THREAD

**Discussion by paowan2** [vernesong/OpenClash#2575] [General]
我订阅了britbox的UK和US服务，虽然都属于britbox，但两地的内容有所不同。

他们都i用到了 licensing.bitmovin.com 这个网站，如果我用UK版的时候需要让访问这个域名走英国的节点，如果我用US版则需要让他走美国节点，手动切换是最笨的方法（参考我下面的示例），有没有能够自动判别走相应节点的办法，规则该怎么写？

=========================================

name: 🖧 我的选择
type: select
proxies:
- 英国节点
- 美国节点
rules:

DOMAIN-SUFFIX,licensing.bitmovin.com,🖧 我的选择

**Comment by vernesong** [comment]
如何判断你在用哪个版本
